src/main/webapp/app/exercises/shared/exercise-detail-common-actions/non-programming-exercise-detail-common-actions.component.html

```diff
@@ -12,19 +12,29 @@
         </a>
 
         <!-- Participations  -->
-        <a *ngIf="exercise.course" [routerLink]="baseResource + 'participations'" class="btn btn-primary btn-sm me-1" style="margin-bottom: 10px">
+        <a
+            *ngIf="exercise.course || exercise.exerciseGroup?.exam?.course"
+            [routerLink]="baseResource + 'participations'"
+            class="btn btn-primary btn-sm me-1"
+            style="margin-bottom: 10px"
+        >
             <fa-icon [icon]="'list-alt'"></fa-icon>
             <span class="d-none d-md-inline" jhiTranslate="artemisApp.exercise.participations">Participations</span>
         </a>
 
         <!--Submissions -->
-        <a *ngIf="exercise.isAtLeastInstructor && exercise.course" [routerLink]="baseResource + 'submissions'" class="btn btn-primary btn-sm me-1" style="margin-bottom: 10px">
+        <a
+            *ngIf="exercise.isAtLeastInstructor && (exercise.course || exercise.exerciseGroup?.exam?.course)"
```

Comment:
could be simplified if `course` would be the input variable

Reply:
👍